Snakefile

```diff
@@ -41,6 +44,38 @@ rule decompress:
         zstd -d -c {input.metadata} > {output.metadata}
         """
 
+
+rule wrangle_metadata:
```

Comment:
I'm a little worried about the use of this rule in this workflow for a couple of reasons. In this case, the wrangle metadata script doesn't address the root issue of the PR which is the presence of duplicate strain records in the incoming data, since switching the "strain" to "accession" allows duplicates to appear in the final build. I would prefer to deduplicate records in the metadata along the lines of [the deduplication we do in the ncov workflow](https://github.com/nextstrain/ncov/blob/b2095c0e502441ebcb0a3054418c9954cf897439/scripts/sanitize_metadata.py#L261) where we prioritize the most recent accession for any duplicated strain names. That same implementation won't likely work here with GenBank accessions, but that general idea would be appropriate here.

The other issue is that this rule is quite complicated and consists mostly of boilerplate code to setup the infrastructure to download a script from another repo. [The script we're downloading](https://github.com/nextstrain/monkeypox/blob/master/scripts/wrangle_metadata.py) is shorter than this rule to download it and the logic performed by the script (renaming and copying columns) could probably be just as easily performed with a couple of csvtk commands. I'm more concerned about this additional complexity because we often reference this Zika workflow as a canonical example of what a Nextstrain workflow should look like and recommend that people use it (or the zika-tutorial) as a template for creating a new pathogen workflow.

Reply:
As an alternative to the wrangle script, we could use the following two csvtk commands:

```
csvtk -t rename -f strain -n strain_original {input.metadata} \
    | csvtk -t mutate -f accession -n strain > {output.metadata}
```

Unlike the wrangle script, the first command will error when the input is missing a "strain" column, but that is standard behavior across Augur and other workflow tools, since we require "strain" to exist in the metadata.

(Copying from our Slack conversation) I can see how deduplication by strain name could be limited to GISAID data where that name has already been curated. If strain name doesn’t make sense as a unique identifier, though, I don’t see its value as a display name. We might just use “accession” as the strain name all through the workflow and into the build. For example, using “Singapore” as the display name for multiple distinct Dengue sequences seems less useful than using the accession.

If we decide that we want "strain" to be the display id in the final Auspice JSON even when we use "accession" as the workflow id, we could add an argument to `augur export v2` along the lines of `--display-strain-field` that would perform the logic currently happening in [the `set_final_strain_name.py` script in the monkeypox workflow](https://github.com/nextstrain/monkeypox/blob/b54768ec17872eb0d898e29527785642f6b98c0d/scripts/set_final_strain_name.py).

Reply:
One more thought about the "display strain field", if we add the following lines to the Auspice config JSON in the "colorings" section:

```json
    {
      "key": "strain_original",
      "title": "strain name",
      "type": "categorical"
    }
```

then we can label tips by strain name through Auspice's dropdown for tip labels as shown in the example below where the URL also updates to reflect that display choice with `/zika?tl=strain_original`.

<img width="1430" alt="image" src="https://user-images.githubusercontent.com/85372/221304254-82c690b8-aca3-43b8-a738-b484ec77dd30.png">